invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -87,7 +113,9 @@ function InVideoQuizXBlock(runtime, element) {
                 if (isProblemToDisplay) {
                   problemToDisplay = $('.xblock-student_view', this)
                   videoState.videoPlayer.pause();
+                  resizeInVideoProblem(problemToDisplay, video);
```

Comment:
as a consequence of changing the `resizeInVideoProblem` function let's make this read: 

```javascript
resizeInVideoProblem(problemToDisplay, video, knownVideoDimensions);
```